lib/services/registered_addons.pm

```diff
@@ -64,8 +64,9 @@ sub check_upgraded_addons {
 }
 
 sub check_suseconnect {
-    my $output = script_output("SUSEConnect -s");
-    my $json   = Mojo::JSON::decode_json($output);
+    my $output = script_output("SUSEConnect -s", 120);
+    my @out    = grep { $_ =~ /identifier/ } split(/\n/, $output);
+    my $json   = Mojo::JSON::decode_json($out[0]);
```

Comment:
Are you sure if the output will always include `identifier`? if there is no `identifier` string in the output, the `@out` will be an empty array.  I do not know how the `decode_json` to deal with an `undef`.

Reply:
Add the condition to confirm the returned value